solenoid_app/views.py

```diff
@@ -6,19 +6,20 @@
 from django.views.generic import TemplateView
 from chartjs.views.lines import BaseLineChartView
 from .forms import DataSetForm, GraphForm
-from solenoid_app.solenoid_math.solver import solenoid_solve, solenoid_range
+from solenoid_app.solenoid_math.solver import solenoid_convert, solenoid_range
+from solenoid_app.solenoid_math.exceptions import *
+from solenoid_app.solenoid_math import ureg
```

Comment:
We could probably do this import in the init file in the solenoid_app directory.  I'm not sure if sharing the same ureg with solenoid_math effect anything but just a thought.

Reply:
I would leave this the way it is. Here is a little snippet I found on SO
"I find it's generally best to avoid importing modules unnecessarily in __init__.py unless you're doing it for namespace reasons. For example, if you want the namespace for your package to look like this"
[https://stackoverflow.com/questions/35727134/module-imports-and-init-py-in-python]